### PR TITLE
* 1692 bug inputtext should not be using textlabelcontrol colours

### LINKED ISSUE #1692: [Bug]: `KryptonMessageBox` text should not be using `TextLabelControl` colours whilst using standard black themes
While investigating #1689, it was found that `InputText` were using `TextLabelControl` colours. Consider using `TextButtonNormal` instead.

### PR DESCRIPTION
**Pull request by Wagnerp**
* Resolves #1692 + a few alterations

image

### CODE REVIEW

**Review by giduac — CHANGES_REQUESTED** [review]
The  introduction of `GlobalStaticMethods` and subsequent changes are not part of this bug and should be handled through a FR.

**Comment by Wagnerp** [comment]
> The introduction of `GlobalStaticMethods` and subsequent changes are not part of this bug and should be handled through a FR.

No, as a bool method is introduced in this class that detects _if_ a black theme is being used. See Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualMessageBoxFormDep.cs, line 166 for usage.

**Comment by giduac** [comment]
> > The introduction of `GlobalStaticMethods` and subsequent changes are not part of this bug and should be handled through a FR.
> 
> No, as a bool method is introduced in this class that detects _if_ a black theme is being used. See Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualMessageBoxFormDep.cs, line 166 for usage.

One simple method is added.
And moving those to a new class is creates a massive change  which is outside the scope of this ticket.

I disagree with those changes.

**Comment by giduac** [comment]
Running: testform/Messagebox test click the top button: creates an error and crashes the program.

**Comment by Wagnerp** [comment]
> > > The introduction of `GlobalStaticMethods` and subsequent changes are not part of this bug and should be handled through a FR.
> > 
> > 
> > No, as a bool method is introduced in this class that detects _if_ a black theme is being used. See Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualMessageBoxFormDep.cs, line 166 for usage.
> 
> One simple method is added. And moving those to a new class is creates a massive change which is outside the scope of this ticket.
> 
> I disagree with those changes.

@giduac 

The 'bigger' changes have been rolled back for a future FR, **but** `IsUsingDefaultBlackThemes()` still remains to avoid code duplication.

**Comment by Wagnerp** [comment]
> Running: testform/Messagebox test click the top button: creates an error and crashes the program.

Hi @giduac 

I don't know why, but it's triggering a error in the `PaletteWrapLabel` class?

**Comment by Wagnerp** [comment]
?

image

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualAboutBoxForm.Designer.cs [hunk 1]
```diff
@@ -28,7 +28,6 @@ protected override void Dispose(bool disposing)
         /// </summary>
```
Comment (giduac) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualAboutBoxForm.Designer.cs:28]: How is the aboutbox related to #1692?

**Comment by giduac** [comment]
Imo this change needs a closer look and there are to many things going on here to just change the text color.
A more simple alteration should be possible to solve this.

**Comment by giduac** [comment]
@Wagnerp 

I have a simple fix for this problem.
As soon as Alpha builds inside VS again I will push the branch for testing.

**Comment by giduac** [comment]
@Wagnerp 
I've tested the simple change and it works fine.
So imo this PR can be Closed.

New PR underway.

**Pull request closed without merging**